README.md

```diff
@@ -53,3 +56,7 @@ The toolchain image build files are kept in [testdata/docker/](testdata/docker/)
 ## License
 
 MIT
+
+### eBPF Gopher
+
+The eBPF honeygopher is based on the Go gopher designed by Renee French.
```

Comment:
Maybe credit Vadim? This means the image is CC BY 3.0 as well?

Reply:
Raised this internally, there are similar open questions in the community at large regarding the eBee mascot and ownership (or not) by the eBPF Foundation and, transitively, the Linux Foundation.

I'll merge this for now, since attribution to Renee is given, but we should publish guidelines on the use of this particular drawing by others in the community. Will adjust later as I receive more info.